samples/net/wpanusb/sample.yaml

```diff
@@ -4,6 +4,6 @@ sample:
 tests:
   test_15_4:
     harness: net
-    platform_exclude: nrf52840_pca10056 usb_kw24d512
+    platform_exclude: nrf52840_pca10056 nrf52840_pca10059 usb_kw24d512
```

Comment:
Should I add this in a separate commit?

Reply:
Yes please

Reply:
@carlescufi Made a separate commit.

Reply:
Why are we blacklisting this sample?  IMHO it's a perfect sample for this device.  And this PR should fix compilation issues: https://github.com/zephyrproject-rtos/zephyr/pull/8912